src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs

```diff
@@ -46,10 +47,30 @@ public Control CreateSettingPanel()
         }
 
         public void Init(PluginInitContext context)
-        {
-            _context = context;
+        {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
             _contextMenuLoader = new ContextMenuLoader(context);
             InitialDriverList();
+
+            _context.API.ThemeChanged += OnThemeChanged;

```

Comment:
I think we need to unregister these events in the dispose function, like we do in the indexer/program plugin.

Reply:
[clarification] Why do we need to do this? The scope of a plugin is for the entire lifecycle of the pt run executable and memory is released at the program termination.

Reply:
You are right in the sense that we don't have to unregister the event handler because the application would terminate, however I was saying it from a futuristic point of view. I think it is better to unregister the event handler as there were some discussions about allowing enabling and disabling individual plugins in the future. This would cause a memory leak then because we aren't releasing managed resources. I leave it to your discretion but since we already have it for the other plugins, I thought it would be good to add it here as well.

Reply:
That's a good point. Thanks for your clarification.